subgroups: update use case group charter (#17)
opiproject/opi: SUBGROUPS.md

## Use Case

* Description & Responsibilities

>   * Create an open channel with the end users and potential deployment partners .

Reviewer: ```suggestion
  * Create an open channel with the end users and potential deployment partners.

```

>   * Share the work being done by the groups and get feedback from the deployment partners .

Reviewer: ```suggestion
  * Share the work being done by the subgroups and get feedback from the end users and deployment partners.

```

>   * Encourage more deployment partners to contribute , take part and join .

Reviewer: ```suggestion
  * Encourage more end users and deployment partners to contribute, take part, and join OPI project.

```

> * Alignment to project goal(s) -- Workgroup leader: Please update this bullet

Reviewer: This section ought to be updated as well. 
```suggestion
* Alignment to project goal(s) -- 
* By engaging with and seeking input from end users and deployment partners, the Use Case subgroup will help by validating the implementation examples and architectures and APIs being worked on by the other subgroups.
*This will also help build the open community to grow and include additional contributorsand members.

```
